experimental/PropertyDDS/packages/property-properties/src/propertyFactory.js

```diff
@@ -697,35 +697,34 @@ class PropertyFactory {
 
 		if (in_compareRemote) {
 			var that = this;
-			this._remoteScopedAndVersionedTemplates.iterate(function (
-				scope,
-				remoteVersionedTemplates,
-			) {
-				if (remoteVersionedTemplates.has(typeidWithoutVersion)) {
-					var previousRemoteVersion = remoteVersionedTemplates
-						.item(typeidWithoutVersion)
-						.getNearestPreviousItem(version);
-
-					if (previousRemoteVersion) {
-						validationResults = that._templateValidator.validate(
-							in_template.serializeCanonical(),
-							previousRemoteVersion.getPropertyTemplate().serializeCanonical(),
-						);
-						warnings.push.apply(warnings, validationResults.warnings);
-					} else {
-						var nextRemoteVersion = remoteVersionedTemplates
+			this._remoteScopedAndVersionedTemplates.iterate(
+				function (scope, remoteVersionedTemplates) {
```

Comment:
Diff is having a hard time with this one.

Changes:
* Moved 'function (' to the next line.
* which increased indentation of the function body
* Moved closing ');' to it's own line [was '});']
* Which introduces a trailing comma to the argument list.

Reply:
Note: if you select the "Hide whitespace" option under the settings (cog icon) menu at the top of the PR (in the github web view), the diff is much easier to follow.